app/views/course/assessment/question/text_responses/_form_comprehension.html.slim

```diff
@@ -0,0 +1,29 @@
+= simple_form_for [current_course, @assessment, @text_response_question] do |f|
+  = f.error_notification
```

Comment:
It failed without any error feedback when I tried to create a questions with one group with one point with one solution. Could you handle the blank `solution_lemma`s? 

In the more general case, could you ensure that unhandled error due to `solution_lemma` are surfaced to the user? Otherwise, it's confusing for the user since the form submission appears to fail for no apparent reason.

Reply:
It failed because the lemmatiser has not been implemented here (it is in #2746).
Will add in an error message when the words could not be lemamatised.

Reply:
I have added the blank `solution_lemma` validation and error message in #2831.